azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -70,6 +70,7 @@
  * Executor manager used to manage the client side job.
  */
 @Singleton
+@Deprecated
```

Comment:
Can you add a comment to specify the class that's replacing this one?

Reply:
added, thanks